deploy/chart/templates/daemonset.yaml

```diff
@@ -33,18 +33,6 @@ spec:
             - --v=3
             - --port=0
             - --secure-port=10253
-            {{- if .Values.linodegoDebug }}
```

Comment:
The `linodego-debug` flag is still present, we should ensure that the helm chart still supports enabling it post-revert.